docs/rest-api-example.md

```diff
@@ -0,0 +1,263 @@
+              'data': [['5c14805d6e502caf61510048192b3835']],
+              'result_kind': 'SUCCESS_WITH_CONTENT'}],
+ 'statement_types': ['INSERT_INTO']}
+'''
+job_id = r.json()['results'][0]['data'][0][0]
+#                            |          |  └--- the 1st column in the 1st result row
+#                            |          └------ the 1st result row
+#                            └----------------- the result of the 1st statement (we'll support executing multiple statements in the future)
+```
+
+Note that executing a `INSERT` statement means to submit it to the Flink cluster. To fetch the result of this statement, we need to do another API call.
+
+```python3
+r = requests.get(host + '/v1/sessions/' + session_id + '/jobs/' + job_id + '/result/0', headers=headers)
+
+pprint.pprint(r.json())
+''' =>
+{'next_result_uri': '/v1/sessions/918e6d1eff0c14228069edec25144022/jobs/5c14805d6e502caf61510048192b3835/result/1',
+ 'results': [{'columns': [{'name': 'affected_row_count',
+                           'type': 'BIGINT NOT NULL'}],
+              'data': [[-2]],
+              'result_kind': 'SUCCESS_WITH_CONTENT'}]}
+'''
+```
+
+The result of a `INSERT` statement will be returned once the insertion finished (be aware that if you're inserting a never-ending stream, this API call will block forever unless the job is canceled).
+
+We currently does not support returning the number of rows inserted, and the `-2` in the result is from JDBC standard, indicating that this operation successes with no information.
+
+## Do some SQL Query
+
+We can now perform SQL queries on our testing table.
+
+```python3
+data = {'statement': 'SELECT * FROM t'}
+# this line may take some time
+r = requests.post(host + '/v1/sessions/' + session_id + '/statements', data=json.dumps(data), headers=headers)
+
+pprint.pprint(r.json())
+''' =>
+{'results': [{'columns': [{'name': 'job_id', 'type': 'VARCHAR(32) NOT NULL'}],
+              'data': [['b1719d4ea8f6d41a830d44918cd7e4d0']],
+              'result_kind': 'SUCCESS_WITH_CONTENT'}],
+ 'statement_types': ['SELECT']}
+'''
+job_id = r.json()['results'][0]['data'][0][0]
+#                            |          |  └--- the 1st column in the 1st result row
+#                            |          └------ the 1st result row
+#                            └----------------- the result of the 1st statement (we'll support executing multiple statements in the future)
+```
+
+We now fetch the results of this query. To illustrate the usage of token, we limit the number of results returned each time with `max_fetch_size`.
+
+```python3
+data = {'max_fetch_size': 3}
+token = 0
+r = requests.get(host + '/v1/sessions/' + session_id + '/jobs/' + job_id + '/result/' + str(token), data=json.dumps(data), headers=headers)
+
+pprint.pprint(r.json())
+''' =>
+{'next_result_uri': '/v1/sessions/8797639f35e1ed71b734c035e03e98da/jobs/b1719d4ea8f6d41a830d44918cd7e4d0/result/1',
```

Comment:
session id should be `918e6d1eff0c14228069edec25144022` (result of first example result)